mango-python/Makefile

```diff
@@ -63,6 +63,9 @@ develop:
 clean_develop:
 	- $(pip) uninstall -y bdgenomics.mango
 	- rm -rf src/*.egg-info
+update_docs:
+	jupyter nbconvert --to notebook --execute ../example-files/notebooks/mango-python-alignment.ipynb --output-dir ../docs/notebooks/
```

Comment:
I am confused, I thought executing these notebooks with a Spark context wasn't working so we were going to add them in manually? The notebooks were not added in this PR, and <make update_docs> is never called.

Reply:
The way the "auto-executing" works now is just that the Mango developer will run `make update_docs` in `mango-python` whenever the example notebooks are updated in `example-files/notebooks`.  This will execute these notebooks and place the resulting notebooks with  output into `docs/notebooks` which are then used when `make html` is run to produce the documentation website.  I just added a note about this in the docs at `docs/installation/source.rst`   The developer will have Spark context available, it was running under Jenkins that was the problem.   

You are right that I I did fail to include the current pre-executed notebooks in `docs/notebooks` in the PR.  These have now been added.